tensorflow/core/kernels/whole_file_read_ops.cc

```diff
@@ -119,4 +119,33 @@ class ReadFileOp : public OpKernel {
 
 REGISTER_KERNEL_BUILDER(Name("ReadFile").Device(DEVICE_CPU), ReadFileOp);
 
+class WriteFileOp : public OpKernel {
+  public:
+    using OpKernel::OpKernel;
+    void Compute(OpKernelContext* context) override {
+	    const Tensor* filename_input;
```

Comment:
can you run clang-format --style=google on all of these files?  That way it'll conform to the rest of the codebase.